.github/workflows/nightly-ci.yml

```diff
@@ -103,44 +105,3 @@ jobs:
       # Important to check version consistency again after bumping versions.
```

Comment:
Curious why we don't merge this consistent version check with the new build file

Reply:
Don't understand. The comment only refers to the first step of the job.

I'm keeping the `nightly-ci` workflow around, so that we still get nightly build/test/... coverage on `main`, but without the publish steps.